java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -4072,6 +4077,115 @@ public T bitfieldReadOnly(
+     * To store the result into a new key, see {@link #sortStore(String, String, SortBaseOptions)}.
+     *
+     * @param key The key of the list, set, or sorted set to be sorted.
+     * @param sortBaseOptions The {@link SortBaseOptions}.
+     * @return Command Response - A <code>Array</code> of sorted elements.
+     */
+    public T sort(@NonNull String key, @NonNull SortBaseOptions sortBaseOptions) {
+        ArgsArray commandArgs = buildArgs(ArrayUtils.addFirst(sortBaseOptions.toArgs(), key));
+        protobufTransaction.addCommands(buildCommand(Sort, commandArgs));
+        return getThis();
+    }
+
+    /**
+     * Sorts the elements in the list, set, or sorted set at <code>key</code> and returns the result.
+     * <br>
+     * The <code>sortReadOnly</code> command can be used to sort elements based on different criteria
+     * and apply transformations on sorted elements.<br>
+     * This command is routed depending on the client's {@link ReadFrom} strategy.
+     *
+     * @since Redis 7.0 and above.
+     * @param key The key of the list, set, or sorted set to be sorted.
+     * @return Command Response - A <code>Array</code> of sorted elements.
+     */
+    public T sortReadOnly(@NonNull String key) {
+        ArgsArray commandArgs = buildArgs(key);
+        protobufTransaction.addCommands(buildCommand(SortReadOnly, commandArgs));
+        return getThis();
+    }
+
+    /**
+     * Sorts the elements in the list, set, or sorted set at <code>key</code> and returns the result.
+     * <br>
+     * The <code>sortReadOnly</code> command can be used to sort elements based on different criteria
+     * and apply transformations on sorted elements.<br>
+     * This command is routed depending on the client's {@link ReadFrom} strategy.
+     *
+     * @since Redis 7.0 and above.
+     * @param key The key of the list, set, or sorted set to be sorted.
+     * @param sortBaseOptions The {@link SortBaseOptions}.
+     * @return Command Response - A <code>Array</code> of sorted elements.
+     */
+    public T sortReadOnly(@NonNull String key, @NonNull SortBaseOptions sortBaseOptions) {
+        ArgsArray commandArgs = buildArgs(ArrayUtils.addFirst(sortBaseOptions.toArgs(), key));
+        protobufTransaction.addCommands(buildCommand(SortReadOnly, commandArgs));
+        return getThis();
+    }
+
+    /**
+     * Sorts the elements in the list, set, or sorted set at <code>key</code> and stores the result in
+     * <code>destination</code>. The <code>sort</code> command can be used to sort elements based on
+     * different criteria, apply transformations on sorted elements, and store the result in a new
+     * key.<br>
+     * To get the sort result without storing it into a key, see {@link #sort(String)} and {@link
+     * #sortReadOnly(String)}.
+     *
+     * @param key The key of the list, set, or sorted set to be sorted.
+     * @param destination The key where the sorted result will be stored.
```

Comment:
very minor, but since earlier you used <code>key</code> , should you continue doing that for the rest of the documentation? specifically at the @param destination part

Reply:
Sorry, I'm not sure what you mean. `destination` is an additional key to the function. Could you elaborate?